Makefile

```
@@ -93,6 +93,7 @@ bundle:
 	mkdir -p dist/$(PLUGIN_ID)
 	cp $(MANIFEST_FILE) dist/$(PLUGIN_ID)/
 	cp assets/logo_dark.png dist/$(PLUGIN_ID)/
+	cp -r i18n/ dist/$(PLUGIN_ID)/
```

```suggestion
	cp -r i18n/ dist/$(PLUGIN_ID)/i18n
```

Original code doesn't create `i18n` directory, so starting plugin is failed.